app/models/pension/AllPensionsData.scala

```diff
@@ -106,8 +108,32 @@ object AllPensionsData {
         pensionSchemeTaxReference = prior.pensionCharges.flatMap(_.pensionSchemeUnauthorisedPayments.map(_.pensionSchemeTaxReference))
       ),
 
-      //TODO Implement PaymentsIntoOverseas Endpoint
-      paymentsIntoOverseasPensions = PaymentsIntoOverseasPensionsViewModel())
+    paymentsIntoOverseasPensions = PaymentsIntoOverseasPensionsViewModel(
```

Comment:
As mentioned during group review, there's much duplication.

The alternative I had in mind was something like this:

```
private def paymentsIntoOverseasPensionsViewModel(prior: AllPensionsData): PaymentsIntoOverseasPensionsViewModel = (prior.pensionIncome, prior.pensionReliefs) match {
    case (Some(pensionsIncome), Some(pensionsReliefs)) => paymentsIntoOverseasPensionsViewModel(pensionsIncome, pensionsReliefs.pensionReliefs)
    case (Some(pensionsIncome), None) => paymentsIntoOverseasPensionsViewModel(pensionsIncome)
    case (None, Some(pensionsReliefs)) => paymentsIntoOverseasPensionsViewModel(pensionsReliefs.pensionReliefs)
    case (None, None) => paymentsIntoOverseasPensionsViewModel()
  }
```

However, given the limited time, best to leave it like this.